src/ports/postgres/modules/utilities/test/pivot.sql_in

```diff
@@ -152,15 +152,19 @@ SELECT assert(__p_7__ = 1.5,
 	'Wrong output in pivoting: Output dictionary') FROM pivout 
 	WHERE id = 0 AND id2 = 0;
 
-DROP AGGREGATE IF EXISTS array_accum (anyelement);
-CREATE AGGREGATE array_accum (anyelement)
-(
-    sfunc = array_append,
+DROP FUNCTION IF EXISTS array_add1(ANYARRAY, ANYELEMENT);
+DROP AGGREGATE IF EXISTS array_accum1 (anyelement);
+CREATE FUNCTION array_add1(ANYARRAY, ANYELEMENT) RETURNS ANYARRAY AS $$
+  SELECT $1 || $2
+$$ LANGUAGE sql STRICT;
```

Comment:
I still appreciate the pointer. This test exists to ensure user defined aggregates work with pivoting in addition to the built-in ones.